india_compliance/gst_india/client_scripts/sales_invoice.js

```diff
@@ -54,6 +54,24 @@ frappe.ui.form.on(DOCTYPE, {
             10
         );
     },
+
+    is_reverse_charge(frm) {
+        if (frm.doc.ecommerce_gstin) {
+            set_supply_liable_to(frm)
+        }
+    },
+    ecommerce_gstin(frm) {
+        if (!frm.doc.ecommerce_gstin) {
+            frm.set_value("supply_liable_to", "")
```

Comment:
Backend logic and hooks for updating field in onload and before_print.(Reger to gst breackup table)